resources/views/fontend/layouts/master.blade.php

```diff
@@ -1,66 +1,101 @@
-  <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/fonts/flaticon/font/flaticon.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/bootstrap.min.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/magnific-popup.css')}}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/jquery.selectBox.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/dropzone.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/rangeslider.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/animate.min.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/leaflet.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/map.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/jquery.mCustomScrollbar.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/fonts/font-awesome/css/font-awesome.min.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/fonts/flaticon/font/flaticon.css') }}">
 
-  <!-- Favicon icon -->
-  <link rel="shortcut icon" href="{{ asset('bower_components/lib_bower/assets/img/favicon.ico') }}" type="image/x-icon" >
+    <!-- Favicon icon -->
+    <link rel="shortcut icon" href="{{ asset('bower_components/lib_bower/assets/img/favicon.ico') }}" type="image/x-icon" >
 
-  <!-- Google fonts -->
-  <link rel="stylesheet" type="text/css" href="{{ asset('bower_components/lib_bower/assets/fonts/font_google.css') }}">
+    <!-- Google fonts -->
+    <link rel="stylesheet" type="text/css" href="{{ asset('bower_components/lib_bower/assets/fonts/font_google.css') }}">
 
-  <!-- Custom Stylesheet -->
-  <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/skins/default.css') }}">
-  <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/style.css') }}">
+    <!-- Custom Stylesheet -->
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/skins/default.css') }}">
+    <link type="text/css" rel="stylesheet" href="{{ asset('bower_components/lib_bower/assets/css/style.css') }}">
+    <script>
+        window.Laravel = <?php echo json_encode([
+            'csrfToken' => csrf_token(),
+        ]); ?>
+    </script>
+
+    <!-- This makes the current users id available in javascript -->
+    @if (!auth()->guest())
+        <script>
+            window.Laravel.userId = <?php echo auth()->user()->id; ?>
```

Comment:
gọi in biến trong laravel blade như nào mà còn dùng echo thế này (facepalm)

Reply:
Ui em lại cứ nghĩ không dùng trong script được nên không thử >< (dance5)